packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx

```diff
@@ -275,6 +275,10 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 		statusComponents.push(containerState.attachState);
 		if (containerState.attachState === AttachState.Attached) {
 			statusComponents.push(connectionStateToString(containerState.connectionState));
+		} else {
+			// If the container is not attached, it is not connected
+			// TODO: review this again(??)
```

Comment:
A little more context about what part you want to review for your TODO would be nice

Reply:
Yeah, we should either make sure this TODO is very clear for other devs reading it, or remove it if it isn't relevant anymore before merging.